packages/sanity/src/form/inputs/files/ImageToolInput/imagetool/HotspotImage.tsx

```diff
@@ -1,6 +1,13 @@
-import React from 'react'
+import React, {
+  memo,
+  startTransition,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react'
 import Debug from 'debug'
-import {debounce} from 'lodash'
```

Comment:
Using `startTransition` instead, combined with 18's `Automatic State Batching` it's no longer necessary to debounce event handlers like `window.onresize`